src/components/balance/BalancePlasm.vue

```diff
@@ -1,19 +1,34 @@
 <template>
   <div v-if="isConnected(currentNetworkStatus)">
-    <div v-if="isH160 || isSS58">
+    <div v-if="isH160">
```

Comment:
![bandicam 2021-11-25 19-11-18-610](https://user-images.githubusercontent.com/40356749/143486722-dd7649a0-5f54-44e1-a2d1-0c10e0cf0f8f.jpg)
![bandicam 2021-11-25 19-11-46-507](https://user-images.githubusercontent.com/40356749/143486753-759337f3-da72-45cb-86a2-fd330d8958d1.jpg)

When I toggle the MetaMask connection and use the H160 account, I can only use the EVM deposit (i.e., my EVM deposit amount is the transferrable balance).  So in this context, the `withdraw` function doesn't make sense. In fact, trying to deposit from it will show an error.
I think we need to improve the UX here.

cc @bLd75

Reply:
Noted.
As far as I know, indeed it doesn't make sense to make a 'EVM withdraw' transfer from MetaMask H160 address. I'll remove the EVM Deposit panel and withdraw button for MetaMask H160 mode.

Reply:
Removed.
![image](https://user-images.githubusercontent.com/92044428/143680682-5cfa9d8a-daaf-4fb0-a007-d6edf1e0df53.png)